discord/component.go

```diff
@@ -508,11 +508,11 @@ func (i *TextInputComponent) MarshalJSON() ([]byte, error) {
 	}
 
 	if i.ValueLimits != [2]int{0, 0} {
```

Comment:
We should rename this to `LengthLimits` then.

Reply:
That's true I missed that. Resolved with https://github.com/diamondburned/arikawa/pull/334/commits/c36309ed0db0dba155ae0830e77a52c15c216a78